### ISSUE TITLE
Custom property to not include note in searches

### ISSUE THREAD

**Issue by kshenoy** [orgzly/orgzly-android#624]
- [x] I have searched for existing issues that may be the same as or related to mine.

Is there a way to specify a custom property to a note/heading to prevent it from showing up in Orgzly's search? eg. I like to keep around the "Getting started with Orgzly" to refer to the examples but I'd prefer not to add `.b."Getting started with Orgzly"` to all of my searches

**Comment by nevenz** [comment]
Would `ARCHIVE` tag do? See #615. It's used by Org mode.

**Comment by kshenoy** [comment]
Hmm, it's not really ARCHIVED per se. I guess it's more similar to what `org-agenda-files` would be in emacs wherein I could open, view and edit any regular org file. However, any items from these files will be included in agenda only if they belong to `org-agenda-files`

**Comment by nevenz** [comment]
That sounds useful and can be implemented, but won't work on individual notes.

**Comment by kshenoy** [comment]
I'm not sure I understand which approach you're referring to. Are you referring to a global setting similar to `org-agenda-files` or the file property? I can understand how the former won't work on individual notes within a file. However, using a property to filter depends on the implementation I suppose i.e. does Orgzly the entire file at once or if it iterates over all the elements in a file to grab items for agenda? 

Anyway, that is not supported even by OrgMode today so it's completely understandable to punt on it :)

**Comment by nevenz** [comment]
> I'm not sure I understand which approach you're referring to. Are you referring to a global setting similar to org-agenda-files or the file property? I can understand how the former won't work on individual notes within a file. 

Yes, I was referring to:

> However, any items from these files will be included in agenda only if they belong to org-agenda-files

> However, using a property to filter depends on the implementation I suppose i.e. does Orgzly slurp the entire file at once or if it iterates over all the elements in a file to grab items for agenda?

It parses org files and keeps everything in database. A different tag or a property could be added, although I tend to avoid adding custom stuff that Org mode doesn't support.

So would supporting a list of notebooks which would be considered for searching (a la `org-agenda-files`) be enough for you? Your "Getting started with Orgzly" suggests it would, but issue's title doesn't.

**Comment by kshenoy** [comment]
> So would supporting a list of notebooks which would be considered for searching (a la org-agenda-files) be enough for you? Your "Getting started with Orgzly" suggests it would, but issue's title doesn't.

Yes, it would. I was thinking of a means of accomplishing that and only while discussing this it occurred to me that using something similar to `org-agenda-files` would also be a way to do that.

Using a property to exclude the entire file or a specific subtree seems more flexible since then you could include the file in `org-agenda-files` and leave out a specific section and I really wish OrgMode supported that. However, till then I agree with your sentiment about not adding custom stuff
